HeapTracing.md

```diff
@@ -0,0 +1,92 @@
+# Heap Tracing
+
+- [Heap Tracing](#heap-tracing)
+  - [Generating Dump File](#generating-dump-file)
+  - [Analyzing the Dump File](#analyzing-the-dump-file)
+    - [Overview Tab](#overview-tab)
+    - [Details Tab](#details-tab)
+
+The heap tracing feature is still under development so please report any found issues on the [github](https://github.com/espressif/idf-eclipse-plugin/issues).
+
+To perform heap tracing, we need to generate a svdat dump file via debugging. The Espressif IDE supports this by adding special breakpoints. To learn more about the configuration at the sdk level and different tracing features please use the following [link](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/system/heap_debug.html).
```

Comment:
Just thinking using Espressif IDE might confuse users since we are not releasing with 2.2.0. You can continue to use the "IDF Eclipse Plugin"